packages/fast-check/jest.setup.js

```diff
@@ -1,5 +1,6 @@
-const process = require('process');
-const fc = require('fast-check');
+import process from 'process';
+import fc from 'fast-check';
```

Comment:
I suspect an issue in Jest with ESM not working the same way in setup files. As if setup files were not sharing the state with the tests themselves. I'll have to confirm that hypothesis on an hello-world example and report it to Jest if confirmed.